ci.sh

```diff
@@ -216,17 +212,22 @@ bundle_qemu() {
 
     mkdir -p "$firmware_target_dir"
     mkdir -p "$qemu_target_dir"
+    if [ -f qemu/pc-bios/edk2-aarch64-code.fd.bz2 ]; then
+      rm -f qemu/pc-bios/edk2-aarch64-code.fd
+      bzip2 -d qemu/pc-bios/edk2-aarch64-code.fd.bz2
+    fi
```

Comment:
@carenas I've started to work on adding support for ARM64 to the GitHub action. For that I'm using the firmware available here [1]. The firmware you're including here (edk2-aarch64-code) takes up 64MB, it's more than all of other files in the archive combined. Does the [1] firmware work for your needs? I rather not include an additional 64MB for something I don't use.

[1] http://releases.linaro.org/components/kernel/uefi-linaro/16.02/release/qemu64/QEMU_EFI.fd

Reply:
There seem to be a lot of bugs with old firmwares (including the one I used), but if it works in your environment then it is likely to also work in others (will test and report back).

Reply:
Thanks.

Reply:
Linaro's firmware doesn't work for my use case because:

* needs to be expanded to 64MB anyway so it can be loaded by QEMU (using either -pflash or an equivalent option)
* it is old, no longer maintained and might be missing important security fixes

Reply:
> needs to be expanded to 64MB anyway so it can be loaded by QEMU (using either -pflash or an equivalent option)

Hmm, it works fine for me without expanding to 64MB with the QEMU `-bios` flag.

> it is old, no longer maintained and might be missing important security fixes

Yeah, that's not good. For some reason I though the latest release was at least from this year, but I see now that it's from 2018. I had problem to login using SSH to an OpenBSD VM using the `edk2-aarch64-code.fd` firmware. Don't know why. The system has managed to boot with the `edk2-aarch64-code.fd` firmware. I'll try again to see if I can make it work.